app/collectors/bucket.scala

```diff
@@ -7,72 +7,96 @@ import conf.AWS
 
   def crawl: Iterable[Bucket] = {
     val request = ListBucketsRequest.builder.build
-    client.listBuckets(request).buckets().asScala
-      .flatMap {
-        Bucket.fromApiData(_, client, origin.awsRegionV2)
-      }
+
+    val listBuckets = client.listBuckets(request).buckets().asScala.toList
+    log.info(s"Total number of buckets with S3 Client region EU-WEST-1 for account ${origin.account} ${listBuckets.length}")
+
+    val listBucketsForCorrectCreatedTime = clientForCorrectCreatedTime.listBuckets(request).buckets.asScala.toList
+    log.info(s"Total number of buckets with S3 Client region US-EAST-1 for account ${origin.account} ${listBucketsForCorrectCreatedTime.length}")
+
+    listBuckets.zip(listBucketsForCorrectCreatedTime).map{ case (bucket, bucketWithCorrectCreatedTime) =>
+      Bucket.fromApiData(bucket, client, origin, bucketWithCorrectCreatedTime)
+    }
   }
 }
 
-object Bucket {
+object Bucket extends Logging {
 
   private def arn(bucketName: String) = s"arn:aws:s3:::$bucketName" 
 
-  def fromApiData(bucket: AWSBucket, client: S3Client, originRegion: Region): Option[Bucket] = {
+  def fromApiData(bucket: AWSBucket, client: S3Client, origin: AmazonOrigin, bucketWithCorrectCreatedTime: AWSBucket): Bucket = {
     val bucketName = bucket.name
-    try {
-      val bucketRegion = client.getBucketLocation(GetBucketLocationRequest.builder.bucket(bucketName).build).locationConstraintAsString
-      if (bucketRegion == originRegion.toString) {
-        Some(Bucket(
-          arn = arn(bucketName),
-          name = bucketName,
-          region = bucketRegion,
-          createdTime = Try(bucket.creationDate).toOption
-        ))
-      } else {
-        None
-      }
+    val bucketRegion = try {
+      Option(
+        client.getBucketLocation(GetBucketLocationRequest.builder.bucket(bucketName).build).locationConstraintAsString
+      )
+        .filterNot(region => "" == region)
+        .orElse(Some(Region.US_EAST_1.id))
     } catch {
-      case e:S3Exception if e.awsErrorDetails.errorCode == "NoSuchBucket" => None
-      case e:S3Exception if e.awsErrorDetails.errorCode == "AuthorizationHeaderMalformed" => None
+      case e:S3Exception if e.awsErrorDetails.errorCode == "NoSuchBucket" =>
+        log.info(s"NoSuchBucket for $bucketName in account ${origin.account}", e)
+        None
+      case e:S3Exception if e.awsErrorDetails.errorCode == "AuthorizationHeaderMalformed" =>
+        log.info(s"AuthorizationHeaderMalformed for $bucketName in account ${origin.account}", e)
+        None
       case NonFatal(t) =>
         throw new IllegalStateException(s"Failed when building info for bucket $bucketName", t)
     }
+    Bucket(
+      arn = arn(bucketName),
+      name = bucketName,
+      region = bucketRegion,
+      createdTime = bucketWithCorrectCreatedTime.creationDate
+    )
   }
 }
 
 case class Bucket(
   arn: String,
   name: String,
-  region: String,
-  createdTime: Option[Instant]
+  region: Option[String],
```

Comment:
As you've mentioned, buckets have an almost pseudo region, it's not the most useful information in this regard. Do we know how clients use this property? Or, to ask in a more direct manner, can we afford to drop this property entirely for buckets?

Reply:
You're right Akash. The region of a bucket has an almost ethereal quality, because it doesn't really matter given that AWS replicate buckets across all available regions. Having said that, I do think it's worth keeping for now, because I'm not sure how other applications are ingesting this field.

Reply:
👍🏽 worth adding a TODO so we can investigate in later work?

Reply:
Good idea!

Reply:
I think we should retain this. It is shown in the S3 dashboard and is helpful to understand where the canonical location of the data it.

Reply:
Thanks for confirming @sihil!👍🏽 Removed TODO in https://github.com/guardian/prism/pull/115